src/index.js

```diff
@@ -0,0 +1,12 @@
+import domMaster from './modules/domMaster';
+import utilities from './modules/utilities';
+
+domMaster.renderForecast();
+
+const searchForm = utilities.getEl('search-form');
+
+searchForm.addEventListener('submit', () => {
+  utilities.getEl('weather-data').innerHTML = '';
```

Comment:
You might want to [prevent the default submit event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event).

```js
searchForm.addEventListener('submit', (e) => {
    e.preventDefault();
  utilities.getEl('weather-data').innerHTML = '';
```